shanoir-ng-front/src/app/breadcrumbs/breadcrumbs.service.ts

```diff
@@ -27,15 +27,14 @@ export class BreadcrumbsService implements OnDestroy {
 
     private popFoundedStepIndex: number;
     public currentStepIndex: number;
-    private nextLabel: string;
-    private nextMilestone: boolean = false;
     private ignoreNavigationEnd: boolean = false;
     private subscriptions: Subscription[] = [];
-    onUpdateSteps: BehaviorSubject<{steps: Step[], operation?: 'ADD' | 'REMOVE' | 'MILESTONE'}> = new BehaviorSubject({steps: this.steps});
+    onUpdateSteps: BehaviorSubject<{steps: Step[], operation?: 'ADD' | 'REMOVE' | 'MILESTONE' | 'FOCUS'}> = new BehaviorSubject({steps: this.steps});
+    private nextPrefill: { field: string; value: any; readOnly: boolean; }[] = [];
```

Comment:
nextPrefill allows to guaranty that the next step will be prefilled as early as possible and avoid bugs with the components initialization cycle that was occuring with the previous method.